apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/Etterbetaling.tsx

```diff
@@ -0,0 +1,117 @@
+import { IEtterbetaling } from '~shared/types/IDetaljertBehandling'
+import { BodyShort, Button, Checkbox, Heading } from '@navikt/ds-react'
+import React, { useEffect, useState } from 'react'
+import { DatoVelger } from '~shared/DatoVelger'
+import styled from 'styled-components'
+import { FlexRow } from '~shared/styled'
+import { isPending, isSuccess, useApiCall } from '~shared/hooks/useApiCall'
+import { lagreEtterbetaling } from '~shared/api/behandling'
+import { hentFunksjonsbrytere } from '~shared/api/feature'
+import { formaterKanskjeStringDato } from '~utils/formattering'
+
+const DatoSection = styled.section`
+  display: grid;
+  gap: 0.5em;
+`
+
+const EtterbetalingWrapper = styled.div`
+  display: flex;
+  gap: 1rem;
+  padding-right: 1rem;
+  margin-top: 1rem;
+  padding-bottom: 2rem;
+`
+
+const Etterbetaling = (props: {
+  behandlingId: string
+  etterbetalingInit: IEtterbetaling | null
+  redigerbar: boolean
+}) => {
+  const { etterbetalingInit, redigerbar, behandlingId } = props
+  const [status, apiLagreEtterbetaling] = useApiCall(lagreEtterbetaling)
+  const [etterbetaling, setEtterbetaling] = useState(etterbetalingInit)
+  const [erEtterbetaling, setErEtterbetaling] = useState<boolean>(!!etterbetaling)
+  const [vis, setVis] = useState(false)
+
+  const [funksjonsbrytere, postHentFunksjonsbrytere] = useApiCall(hentFunksjonsbrytere)
+  const featureToggleNameEtterbetaling = 'registrer-etterbetaling'
+
+  useEffect(() => {
```

Comment:
On a side note: Nesten så man skulle hatt en wrapper komponent for feature toggling hvis ikke noe annet skal vises hvis den er false.

Reply:
Einig! Dette er litt knotete greier å ha alle stadar vi skal toggle noko.